docs/index.md

```diff
@@ -11,22 +11,32 @@ The SMPKit helps you build services that execute investment strategies on liquid
 
 	This section is under active development - please report issues in the [Autonolas Discord](https://discord.com/invite/z2PT65jKqQ).
 
-Once you have {{set_up_system}} to work with the Open Autonomy framework, you can run a local demo based on the SMPKit as follows:
+In order to run a local demo based on the SMPKit:
 
-1. Fetch the Smart Managed Pools service.
+1. [Set up your system](https://docs.autonolas.network/open-autonomy/guides/set_up/) to work with the Open Autonomy framework. We recommend that you use these commands:
+
+    ```bash
+    mkdir your_workspace && cd your_workspace
+    touch Pipfile && pipenv --python 3.10 && pipenv shell
+
+    pipenv install open-autonomy[all]==0.9.0
```

Comment:
Why use `Pipenv` just to install a single package and run a single demo?

Reply:
Addressed on https://github.com/valory-xyz/apy-oracle/pull/45#discussion_r1115496043.